test/include/sigv4_config.h

```diff
@@ -30,12 +30,30 @@
 
 #include <stdio.h>
 
-#define LogError( message )    printf( "Error: " ); printf message; printf( "\n" )
+#ifdef DISABLE_LOGGING
```

Comment:
Why do we need this if we are using NDEBUG as well?
Does this serve some different purpose?